app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
```

Comment:
Bad import style. It's better to use [grouping mechanism](https://peps.python.org/pep-0328/#rationale-for-parentheses) in that case.